.editorconfig

```diff
@@ -24,3 +24,39 @@ indent_size = 2
 # JSON files
 [*.json]
 indent_size = 2
+
+# Dotnet code style settings:
+[*.{cs, vb}]
+# Avoid "this." unless absolutely necessary
```

Comment:
yes just realized i need to add the "Me." part. @Pilchie do I just add this and push again?